README.md

```diff
@@ -1,2 +1,71 @@
-# module-ballerina-random
 Ballerina Random Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-random/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-random/actions?query=workflow%3ABuild)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-random.svg)](https://github.com/ballerina-platform/module-ballerina-random/commits/main)
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/random.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Frandom)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The Random library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This module provides functions related to random number generation.
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/).
```

Comment:
```suggestion
```

Reply:
Shall we remove this. since we don't have a BBE to point. will add this when we add a BBE

Reply:
Ack. Temporarily removed.